Src/AutoMoq/MockVirtualMethodsCommand.cs

```diff
@@ -115,6 +115,13 @@ private static IEnumerable<MethodInfo> GetConfigurableMethods(Type type)
                               ? type.GetInterfaceMethods()
                               : type.GetMethods();
 
+            // Don't mock "GetSet" properties because they will be stubbed.
+            var getSetMethods = type.GetProperties()
+                                    .Where(p => p.GetGetMethod() != null &&
+                                                p.GetSetMethod() != null)
+                                    .SelectMany(p => new[] {p.GetGetMethod(), p.GetSetMethod()});
+            methods = methods.Where(m => !getSetMethods.Contains(m));
```

Comment:
This implementation isn't very optimal as expensive enumeration will happen for each time you call for `Contains`. It's better to use the `Except()` LINQ method for such cases.